### ISSUE TITLE
`.mov` vs `.qt`

### ISSUE THREAD

**Issue by shirotech** [broofa/mime#277]
Why is `.qt` preferred over `.mov` for `video/quicktime`, did a quick google lands here: https://discussions.apple.com/thread/951061#:~:text=No%20difference%20except%20the%20file,be%20understood%20by%20Web%20servers.

> No difference except the file extension. The .mov extension should be used as .qt is very old and may not be understood by Web servers.

**Comment by broofa** [comment]
`getExtension()` returns the first extension in the `extensions` array found in mime-db, [here](https://github.com/jshttp/mime-db/blob/master/db.json#L8374).

**Comment by shirotech** [comment]
Yes but does not answer the question, so who or what decides which one should go first?

**Comment by broofa** [comment]
See https://github.com/jshttp/mime-db/issues/86

tl;dr: See if you can convince the Apache project to either reorder the quicktime extension in their list, or remove the `qt` extension.
